Entities/AHuman.h

```diff
@@ -959,6 +994,7 @@ ClassInfoGetters;
     float m_JetTimeTotal;
     // How much time left the jetpack can go, in ms
     float m_JetTimeLeft;
+	float m_JetReplenishRate; //!< How fast the jetpack fuel will replenish when not in use, in relation to the legacy value.
```

Comment:
All jet replenish rate comments from acrab apply to ahuman too of course. This is just a reminder :)